code/game/objects/items/food/misc.dm

```diff
@@ -499,10 +505,14 @@
 		return
 	if(prob(15))
 		new /datum/hallucination/oh_yeah(victim)
-		to_chat(victim, span_colossus("<b>[pick("I AM IMMORTAL.", "I SHALL TAKE YOUR WORLD.", "I SEE YOU.", "YOU CANNOT ESCAPE ME FOREVER.", "NOTHING CAN HOLD ME.")]</b>"))
+		to_chat(victim, span_colossus("<b>[pick(hallucination_lines)]</b>"))
 	else
 		to_chat(victim, span_warning("[pick("You hear faint whispers.", "You smell ash.", "You feel hot.", "You hear a roar in the distance.")]"))
 
+/obj/item/food/bubblegum/bubblegum/suicide_act(mob/user, show_message=FALSE)
+	. = ..()
```

Comment:
You should probably move this to the end with `return ..()`, since `suicide_act` might kill them first.

Reply:
suicide_act() doesn't handle the suicide itself, the verb does that. suicide_act() is just for extra stuff you wanna put, but I'm gonna change this anyways